storybook/pages/OnboardingLayoutPage.qml

```diff
@@ -326,14 +324,14 @@ SplitView {
 
     BiometricsPopup {
         id: biometricsPopup
-        visible: onboarding.stack.currentItem instanceof LoginScreen && ctrlBiometrics.checked && ctrlTouchIdUser.checked
+
```

Comment:
Here the popup is stealing the input; and even worse, it's freezing the UI completely:

![image](https://github.com/user-attachments/assets/97a53145-2e76-43ea-a3b5-f2a8d087eadf)

It should be "above" but w/o focus

Reply:
When the fingerprint is not recognized, this is what the user should see:

![image](https://github.com/user-attachments/assets/b5802b3f-65cb-4540-bf7c-ea1c8d1ab03a)

Reply:
A failed, red state, and an option to use the password instead

Reply:
In OnboardingLayout, I pressed "Simulate correct fingerprint", it succeeded but the PIN is not filled in, nor the "login" happens:

![image](https://github.com/user-attachments/assets/b4dd1f56-a408-4ca0-a209-f7a5de88a8b7)

The very same scenario works fine when invoked from the LoginScreen SB page

Reply:
As mentioned above, I did it intentionally but we can discuss pros/cons and adjust.

Reply:
> A failed, red state, and an option to use the password instead
Afaik it's not possible to reach such state in practice. User may switch from fingerprint to password withing the mac's dialog but app is not informed about that, so there is no way to change color.

Reply:
> In OnboardingLayout, I pressed "Simulate correct fingerprint", it succeeded but the PIN is not filled in, nor the "login" happens:

It was old regression there. Fixed.